### DISCUSSION TITLE
Copilot "on-demand" as opposed to always spamming

### DISCUSSION THREAD

**Discussion by John-Colvin** [zed-industries/zed#15678] [Feature Requests]
I find that copilot suggestions are pretty distracting when I'm actually trying to think, but very useful for "fill out the rest here".

I would love to be able to have the suggestions off, but then trigger a suggestion with a keyboard shortcut.

Is there a way to do this?

`"show_inline_completions": true` means I get the spam all the time, but if I set it to `false` then even if I run `editor: show inline completion" nothing appears (it's not a keyboard shortcut problem).

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
